web/src/common/ui/responsive-image/ResponsiveImage.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
```

Comment:
Let's implement this component as well rather than merging something unfinished.